metadata.json

```diff
@@ -56,7 +56,7 @@
     {"name":"zack/r10k",                      "version_requirement":">= 2.3.0"},
 
     {"name":"puppetlabs/acl",                 "version_requirement":">= 1.0.0"},
-    {"name":"puppetlabs/dsc",                 "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/dsc",                 "version_requirement":"1.2.0"},
```

Comment:
Yes, I tested the PI labs with these two versions, and that seems to fix the agent hang problem. There's still some other issue with the userprefs lab, though, but it may be a chocolatey problem. It can't install any of the Windows editors for some reason.